### PR TITLE
covid-data-app: final

### PR DESCRIPTION
**Pull request by Melaku05**
added  live presentation video on read-me file
 added Live demo on read-me file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,12 @@
 -This is a mobile web application that displays a list of Covid Data generated for the world country,and  to check numaric details of Covid-19 in countries from an API.
 -This API provides data on COVID-19.
```
Comment (KDlamini) [inline README.md:7]: In the "Acknowledgements" section or similar on the README, please give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Melaku05 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating a working Covid Metrics app. 🎊
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Nice touch with the interactive search bar 👏🏾 
✔️  Impressive use of loading UI when API is still fetching data.
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Melaku05) [reply README.md:7]: Thank you, I did that

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
-
+- Inspired by original design for Creative Commons by Nelson Sakwa on [Behance](<https://behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)>)
```
Comment (mateo951) [inline README.md:72]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @Melaku05 , 

In my opinion, your project is ready for the presentation! There is nothing else to say other than... it's time to merge it 🤓 

### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**